### PR TITLE
Add doc on confidential

### PR DESCRIPTION
**Pull request by olethanh**
Sor

### CODE REVIEW

File: docs/computing/confidential.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Aleph.im allow to launch confidential VM, securised with AMD SEV. This is also known as TEE, Trusted Execution Environment.
+
+The disk and the memory are fully encrypted so no one can see what happen inside your VM.
+
+At the moment they can only be launched via the aleph client cli tool.
+
+
+# Requirements:
+ * The sevctl command tool
+ * The aleph cli client, properly set up with an Ethereum account
```
Comment (Psycojoker) [inline docs/computing/confidential.md:12]:
```suggestion
 * The [aleph cli client](https://github.com/aleph-im/aleph-client), properly set up with an Ethereum account
```

**Comment by hoh** [comment]
This doc needs to refer how to install aleph-client : https://docs.aleph.im/tools/aleph-client/#requirements

File: docs/computing/confidential.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ * A ssh key
+
+# Guide
+1. Create a confidential disk image
+You must create a virtual machine disk image,  encrypted with a password of your choice.
+
+Follow the instruction and scripts provided here: https://github.com/aleph-im/aleph-vm/blob/main/examples/example_confidential_image/README.md
```
Comment (hoh) [inline docs/computing/confidential.md:19]:
```suggestion
Follow the instruction and scripts provided here: [https://github.com/aleph-im/aleph-vm/blob/main/examples/example_confidential_image/README.md](https://github.com/aleph-im/aleph-vm/blob/main/examples/example_confidential_image/README.md)
```

File: docs/computing/confidential.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Aleph.im allow to launch confidential VM, securised with AMD SEV. This is also known as TEE, Trusted Execution Environment.
+
+The disk and the memory are fully encrypted so no one can see what happen inside your VM.
+
+At the moment they can only be launched via the aleph client cli tool.
+
+
+# Requirements:
+ * The sevctl command tool
```
Comment (hoh) [inline docs/computing/confidential.md:11]: How do I install these software ?

**Comment by olethanh** [comment]
@hoh Following our discussion I checked and your command to upload to ipfs  works for me. Not sure what your issue was?

**Comment by olethanh** [comment]
@hoh You copied the encrypted image creation documentation but not the attached scripts, so the doc don't really make sense anymore.

I don't really feel aleph-docs is the place to have shell script but how do you suggest we fix this?

File: docs/computing/confidential/instance.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## 0. Ensure you have uploaded your VM in the aleph network.
+See previous section: [Encrypted Virtual Machine image](./encrypted-disk.md)
+
+## 1. Create your confidential Instance in Aleph.
+
+This will ask you how much CPU, RAM and Disk you want to use and on which node (CRN) to deploy it.
+
+```shell
+aleph instance create --confidential --crn-url https://aleph8.agot.be/
```
Comment (hoh) [inline docs/computing/confidential/instance.md:13]: Replace this line

**Comment by Psycojoker** [comment]
This should be good now (Hugo has review it), we'll wait for the announcement before merging it.

This can be used as an internal reference for testing.

**Review by philogicae — APPROVED** [review]
Checked and tested (following all the steps). All good imo

**Pull request merged**